ingestion-sink/src/main/java/com/mozilla/telemetry/ingestion/sink/io/BigQuery.java

```diff
@@ -1,10 +1,12 @@
 package com.mozilla.telemetry.ingestion.sink.io;
 
+import avro.shaded.com.google.common.collect.ImmutableList;
```

Comment:
We should avoid using avro's shaded guava.

Reply:
>_< nice catch